_extras/snippets_library/UCL_sge/13/long_job_err.snip

```diff
@@ -1 +1,2 @@
-=>> PBS: job killed: walltime 33 exceeded limit 30
+This script is running on:
+node-d00a-007.myriad.ucl.ac.uk
```

Comment:
Is this what's shown when a job fails and gets killed?

Reply:
With SGE on Myriad we get no error when the job times out and gets killed, unlike with other schedulers. That's why I added the 'finished successfully' line to the example scripts. I would say these changes are specific to our lesson and not necessary to propagate upstream.